client/src/containers/CreateCardFormContainer/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react'
+import CreateCardForm from 'src/components/CreateCardForm'
+
+const axios = require('axios')
+
+class CreateCardFormContainer extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      description: '',
+    }
+  }
```

Comment:
Simpler format can be used:
```js
state = {
  description: '',
}
```

instead of:

```js
constructor(props) {
  super(props)
  this.state = {
    description: '',
  }
}
```

Reply:
👌